k8s/namespaces/unspec/unspec-service/demo.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: helm.fluxcd.io/v1
```

Comment:
your patches need referencing in your environment specific overlay,
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/common-overlay/rpe/kustomization.yaml#L6-L7